tutorials/dataframe/df000_simple.C

```diff
@@ -0,0 +1,26 @@
+/// \file
+/// \ingroup tutorial_dataframe
+/// Simple RDataFrame example in C++.
+///
+/// This tutorial shows a minimal example of RDataFrame that creates an
+/// empty-source data frame, adds a new column `x` with random numbers
+/// and finally creates and draws a histogram for `x`.
```

Comment:
I don't find "empty-source" self-explanatory. Maybe
 
```suggestion
/// This tutorial shows a minimal example of RDataFrame. It starts without input
/// data, generates a new column `x` with random numbers, and finally draws
/// a histogram for `x`.
```